cirq/sim/simulator.py

```diff
@@ -532,3 +532,7 @@ def _value_equality_values_(self):
         measurements = {k: v.tolist() for k, v in
                         sorted(self.measurements.items())}
         return (self.params, measurements, self._final_simulator_state)
+
+    @property
+    def qubit_map(self):
```

Comment:
Add a docstring explaining how this would be used, and also a return type annotation.

Reply:
done